cmd/juju/model/exportbundle_test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for info.
+
+package model_test
+
+import (
+	"github.com/juju/cmd/cmdtesting"
+	"github.com/juju/errors"
+	jujutesting "github.com/juju/testing"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/cmd/juju/model"
+	coremodel "github.com/juju/juju/core/model"
+	"github.com/juju/juju/jujuclient"
+	"github.com/juju/juju/testing"
+)
+
+type ExportBundleCommandSuite struct {
+	testing.FakeJujuXDGDataHomeSuite
+	fake  *fakeExportBundleClient
+	stub  *jujutesting.Stub
+	store *jujuclient.MemStore
+}
+
+var _ = gc.Suite(&ExportBundleCommandSuite{})
+
+func (f *fakeExportBundleClient) Close() error { return nil }
```

Comment:
Please move all these fakeExportBundleClient function next to the struct declaration, at the end of the file.

Reply:
done